base/gmp.jl

```diff
@@ -334,7 +334,7 @@ for (fJ, fC) in ((:-, :neg), (:~, :com))
     end
 end
 
-function <<(x::BigInt, c::Int32)
+function <<(x::BigInt, c::Int)
```

Comment:
The automatic conversion with range checking is nice like that :-)

This is slightly inefficient, but since this is BigInt code, that's the least of the performance concerns.